plugins/woocommerce/changelog/55519-enhancement-52679

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+#52679 - Allow filtering of `comment_row_actions` in Review List Table
```

Comment:
```suggestion
Allow filtering of `comment_row_actions` in Review List Table
```
We will automatically add PR number when releasing the next version so we don't need to add it manually.